sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -525,9 +525,9 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
     .createWithDefault(false)
 
   val POOLED_MEM = conf("spark.rapids.memory.gpu.pooling.enabled")
```

Comment:
Can we please just delete this config entirely? It has been deprecated for over 5 years. No one should be using it any more.

Reply:
Done